### PR TITLE
update min timeout value for seconds

### PR DESCRIPTION
**Pull request by AceTugboat**
current min time out is in miliseconds, we want that to be in seconds.

updated the defauld value for the timeout to 12s because today is 12/12 :)

### CODE REVIEW

File: caliper/base.py [hunk 1]
```diff
@@ -182,11 +182,11 @@ def CONNECTION_TIMEOUT(self):
 
     @CONNECTION_TIMEOUT.setter
     def CONNECTION_TIMEOUT(self, new_timeout):
-        if int(new_timeout) >= 1000:
+        if int(new_timeout) >= 1:
             self._config['CONNECTION_TIMEOUT'] = int(new_timeout)
         else:
             raise_with_traceback(
-                ValueError('new timeout value must be at least 1000 milliseconds'))
+                ValueError('new timeout value must be at least 1 second'))
```
Comment (dcgleim) [inline caliper/base.py]: I wonder why IMS thought this was necessary validation.  And all for an option they never actually used anywhere.

**Comment by dcgleim** [comment]
wait... shouldn't we make it 1 since today is 12/12?

File: caliper/base.py (hunk 1 above)
Comment (AceTugboat) [reply caliper/base.py]: and in a unit that the Requests module doesn't even expect natively. 🤷‍♂

**Comment by AceTugboat** [comment]
I was thinking maybe 2019/12/12 = 14.020833

**Comment by dcgleim** [comment]
ha good call!

**Pull request merged**